qiskit_experiments/library/stark/ramsey_xy.py

```diff
@@ -0,0 +1,319 @@
+    # section: note
+
+        In the lower power limit, the Stark shift has a quadratic dependency on the
+        pulse amplitude. This implies the resulting Stark shift is not sensitive to
+        the sign of the amplitude of StarkV and StarkU pulses on one hand.
+        On the other hand, the sign of the shift depends on the sign of frequency offset.
+        In other words, positive (negative) Stark shift occurs when the Stark tone frequency
+        is lower (higher) than the qubit frequency.
+
+        In this experiment, the amplitude of Stark tone, i.e. `stark_amp` parameter, is
+        abstracted to give positive (positive) Stark shift with positive (negative) stark_amp
+        for better handling of the experiment parameters.
+        The StarkV and StarkU pulses are always positive regardless of the sign of stark_amp,
+        while the sign of frequency offset is implicitly determined by its sign.
+
+    # section: analysis_ref
+        :py:class:`RamseyXYAnalysis`
+
+    # section: see_also
+        qiskit_experiments.library.characterization.ramsey_xy.RamseyXY
+
+    """
+
+    def __init__(
+        self,
+        qubit: int,
+        stark_amp: float,
+        backend: Optional[Backend] = None,
+        **experiment_options,
+    ):
+        """Create new experiment.
+
+        .. note::
+
+            .. _stark_amplitude:
+
+            The `stark_channel` in the experiment options may be necessary supplied.
+            By default, the Stark tone is applied to the same channel with the qubit drive
+            with frequency modulation. This frequency modulation might update the
+            channel frame, which accumulates unwanted phase against the frequency difference
+            between the qubit drive and Stark tone frequency in addition to
+            the qubit Stark shift. You can use a dedicated Stark drive channel if available,
+            otherwise you may want to reuse a control channel associated with the qubit.
+            This depends on the architecture of your quantum device.
+
+        Args:
+            qubit: Index of qubit.
+            stark_amp: A single float parameter to represent the magnitude of Stark tone
+                and the sign of expected Stark shift. See :ref:`stark_amplitude` for details.
+            backend: Optional, the backend to run the experiment on.
+            experiment_options: Extra experiment options. See self.experiment_options.
+        """
+        self._timing = None
+
+        super().__init__(qubits=[qubit], analysis=RamseyXYAnalysis(), backend=backend)
+        self.set_experiment_options(stark_amp=stark_amp, **experiment_options)
+
+    @classmethod
+    def _default_experiment_options(cls) -> Options:
+        """Default experiment options.
+
+        Experiment Options:
+            stark_amp (float): A single float parameter to represent the magnitude of Stark tone
+                and the sign of expected Stark shift. See :ref:`stark_amplitude` for details.
```

Comment:
I created a dedicated tutorial in [afd62a8](https://github.com/Qiskit/qiskit-experiments/pull/1008/commits/afd62a804490a4fd5e6b4527d732a755ce4a6ee9) because this is likely referred by every Stark experiments.